pkg/util/trace_info.go

```diff
@@ -161,7 +162,7 @@ func (t *TraceInfo) generateRandomString() string {
 	for i := range s {
 		s[i] = letters[t.r.Intn(len(letters))]
 	}
-	return string(s)
+	return "vulture-" + string(s)
```

Comment:
While we're here should we take a minute to think about whether we need completely random strings or if we should cap them to some cardinality?

We could set it to a 1000. And the span names could be `vulture-0`, `vulture-1`, ..., `vulture-999` (again this is similar to what we do with tags)

Reply:
Yeah, that's fair. Span names aren't expected to be super high cardinality. Depends a bit on whether we like testing search with this abnormal high cardinality or not.
`generateRandomString` is also used to generate tag values, so we might have to split them up.